sycl/unittests/Extensions/CommandGraph.cpp

```diff
@@ -282,6 +286,10 @@ TEST_F(CommandGraphTest, SubGraph) {
   ASSERT_EQ(sycl::detail::getSyclObjImpl(Node2MainGraph)->MPredecessors.size(),
             1lu);
 
+  // We need to disable backend for unitests as backends are not loaded
+  auto MainGraphImpl = sycl::detail::getSyclObjImpl(MainGraph);
+  MainGraphImpl->setEmulationModeForced(true);
+
```

Comment:
Could we create a helper function for this to not have to repeat this code?

Reply:
Did it